extensions/cql/standard/clause_6_filter.adoc

```diff
@@ -7,29 +7,29 @@ include::requirements/requirements_class_filter.adoc[]
 
 <<OAFeat-1,OGC API - Features - Part 1: Core>> (and the draft http://docs.opengeospatial.org/DRAFTS/20-024.html[OGC API - Common - Part 2: Geospatial Data] standard) define two filtering parameters on the resource at path `/collections/{collectionId}/items`: `bbox` and `datetime`. <<OAFeat-1,OGC API - Features - Part 1: Core>> also adds support for simple equality predicates logically joined using the `AND` operator. These capabilities offer simple resource filtering for HTTP requests.
 
-This requirements class defines additional query parameters that allow more
+This Filter class defines additional query parameters that allow more
```

Comment:
This should be: "The Filter requirements class ..."

Reply:
Fixed.